Source/Core/Core/Boot/Boot.h

```diff
@@ -52,5 +52,5 @@ class CBoot
   static bool Load_BS2(const std::string& _rBootROMFilename);
   static void Load_FST(bool _bIsWii);
 
-  static bool SetupWiiMemory();
+  static bool SetupWiiMemory(u64 ios_version);
```

Comment:
ios_title_id? version would be more like 80.

Another option is to actually call this ios_version but make it a u8 :)